src/main/java/uk/gov/pay/api/resources/GetPaymentRefundsStrategy.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.api.resources;
+
+
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.RefundsResponse;
+import uk.gov.pay.api.service.GetPaymentRefundsService;
+
+public class GetPaymentRefundsStrategy extends LedgerOrConnectorStrategyTemplate<RefundsResponse> {
+
+    private final Account account;
+    private final String paymentId;
+    private final GetPaymentRefundsService getPaymentRefundsService;
+
+    public GetPaymentRefundsStrategy(String strategy, Account account, String paymentId,
+                                     GetPaymentRefundsService getPaymentRefundsService) {
+        super(strategy);
+        this.account = account;
+        this.paymentId = paymentId;
+        this.getPaymentRefundsService = getPaymentRefundsService;
+    }
+
+    @Override
+    protected RefundsResponse executeLedgerOnlyStrategy() {
+        return getPaymentRefundsService.getLedgerTransactionTransactions(account, paymentId);
+    }
+
+    @Override
+    protected RefundsResponse executeFutureBehaviourStrategy() {
+        return getPaymentRefundsService.getLedgerTransactionTransactions(account, paymentId);
```

Comment:
might be more obvious while reading the code if it was:
```suggestion
        return executeLedgerOnlyStrategy();
```

Reply:
Updated!